x-pack/plugins/remote_clusters/public/sections/remote_cluster_form/remote_cluster_form.js

```diff
@@ -417,6 +538,32 @@ export class RemoteClusterFormUi extends Component {
     return null;
   }
 
+  renderErrors = () => {
+    const { areErrorsVisible, fieldsErrors } = this.state;
+    const errorValues = Object.values(fieldsErrors);
+    const hasErrors = errorValues.some(error => error !== undefined);
+
+    if (!areErrorsVisible || !hasErrors) {
```

Comment:
This works great. What do you think about setting the `areErrorsVisible` to false in `onFieldsChange()`?

```js
this.setState({
      fields: newFields,
      fieldsErrors: this.getFieldsErrors(newFields),
      areErrorsVisible: false, // Add this?
    });
```

Reply:
I tried it and it introduces a couple problems to the UX:

1. Errors disappear as soon as you start typing, so you don't know what the errors in the other fields are
2. You don't know if you've addressed your error until you hit "Save" again to re-validate everything

Reply:
True, did not have 2 errors on the screen when testing this.